doc/integrations/duckdb.md

```diff
@@ -249,3 +249,21 @@ SELECT *
 FROM df
 WHERE x > 95
 ```
+
+## Passing parameters to connection
+```{code-cell} ipython3
+from sqlalchemy import create_engine
+some_engine = create_engine(
+    'duckdb:///:memory:',
+    connect_args={
+        'preload_extensions': ['https'],
```

Comment:
```suggestion
        'preload_extensions': ['httpfs'],
```

Reply:
thanks
am closing this pull request and creating one again

Reply:
you can push to the same branch and it'll update the pull request

Reply:
Not sure why you changed it back, httpfs is the correct name - though you will need to install it first